src/v3/src/transformer/identify/transformIdentify.ts

```diff
@@ -38,25 +40,19 @@ export const transformIdentify: IdxStepTransformer = (_, formBag, widgetProps) =
     'credentials.passcode',
     uischema.elements as UISchemaElement[],
   )) {
-    submitBtnElement.label = 'oie.primaryauth.submit';
+    submitBtnElement.label = loc('oie.primaryauth.submit', 'login');
   }
 
-  const rememberMe = getUIElementWithName(
-    'rememberMe',
-    uischema.elements as UISchemaElement[],
-  );
   if (features?.showKeepMeSignedIn === false) {
     uischema.elements = removeUIElementWithName(
       'rememberMe',
       uischema.elements as UISchemaElement[],
     );
-  } else if (rememberMe) {
-    rememberMe.label = 'oie.remember';
```

Comment:
Is this being removed because it's already there in the response?

Reply:
Somewhat, so there is now a top level function that handles transformation of all element labels (similar to what v2 does with its i18nTransformer). Since that handles transforming all labels, this is no longer needed.